doc/Programs/03-environment-variables.pod6

```diff
@@ -44,70 +44,70 @@ version 6.e.  Early implementation has been available in Rakudo compiler
 as of version 2019.12, and before that it was available as
 C<PERL6_EXCEPTIONS_HANDLER>.
 
-X<|RAKUDO_NO_DEPRECATIONS>
+X<|Programs,RAKUDO_NO_DEPRECATIONS>
 =item C<RAKUDO_NO_DEPRECATIONS> (I<Bool>; B<src/core.c/Deprecations.pm6>)
 
 If true, suppresses deprecation warnings triggered by the C<is DEPRECATED>
 trait.
 
-X<|RAKUDO_DEPRECATIONS_FATAL>
+X<|Programs,RAKUDO_DEPRECATIONS_FATAL>
 =item C<RAKUDO_DEPRECATIONS_FATAL> (I<Bool>; B<src/core.c/Deprecations.pm6>)
 
 If true, deprecation warnings become thrown exceptions.
 
-X<|RAKUDO_VERBOSE_STACKFRAME>
+X<|Programs,RAKUDO_VERBOSE_STACKFRAME>
 =item C<RAKUDO_VERBOSE_STACKFRAME> (I<UInt>; B<src/core.c/Backtrace.pm6>)
 
 Displays source code in stack frames surrounded by the specified number of
 lines of context; for instance C<RAKUDO_VERBOSE_STACKFRAME = 1> will use one
 context line.
 
-X<|RAKUDO_BACKTRACE_SETTING>
+X<|Programs,RAKUDO_BACKTRACE_SETTING>
 =item C<RAKUDO_BACKTRACE_SETTING> (I<Bool>; B<src/core.c/Backtrace.pm6>)
 
 Controls whether C<.setting> files are included in backtraces.
 
 =head2 Affecting precompilation
 
-X<|RAKUDO_PREFIX>
+X<|Programs,RAKUDO_PREFIX>
```

Comment:
Not a program. Neither of these is a program. It's an environment variable.

Reply:
We have types of pages like `type`, `language` and also `programs`.

Not sure if it was intended, but `programs` ended up to relate to everything "near" the programming process, like how to debug Raku programs or env variables for the compiler.

I am not sure what'd be a better term for this.

Reply:
As long as we're introducing or changing categories, why not go the whole nine yards? I'd say "Environment variables" would be simply it.

Reply:
I was using the list we agreed upon in https://github.com/Raku/problem-solving/pull/271/files with your approval, but yes, can be changed in theory.

Reply:
Totally different to seeing it applied to specific terms.